node/validator_store.go

```diff
@@ -270,7 +277,12 @@ func NewValidatorStore(
 			return nil, fmt.Errorf("failed to get chunks table: %w", err)
 		}
 
-		err = chunkTable.SetCacheSize(uint64(config.LittDBChunkCacheSizeGB * units.GiB))
+		err = chunkTable.SetWriteCacheSize(uint64(config.LittDBWriteCacheSizeGB * units.GiB))
+		if err != nil {
+			return nil, fmt.Errorf("failed to set cache size for chunks table: %w", err)
```

Comment:
error message should specify setting write cache size failed

Reply:
fixed